app/controllers/gobierto_participation/issues_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module GobiertoParticipation
+  class IssuesController < GobiertoParticipation::ApplicationController
+    def index
+      @issues = current_site.issues.alphabetically_sorted
+    end
+
+    def show
+      @issue = find_issue
+      @issue_news = find_issue_news
+      @issue_news_updated = find_issue_news_updated
+      @issue_events = find_issue_events
+      @processes = current_site.processes.process.open
```

Comment:
No, I forgot it

https://github.com/PopulateTools/gobierto/pull/868/commits/f1c3ee240e5bb4930543e445d9033a03be6d8126